command/agent_monitor_export.go

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/cli"
+	"github.com/hashicorp/nomad/api"
+	"github.com/posener/complete"
+)
+
+type MonitorExportCommand struct {
+	Meta
+
+	// Below this point is where CLI flag options are stored.
+	nodeID      string
+	serverID    string
+	onDisk      bool
+	logSince    time.Duration
+	serviceName string
+	follow      bool
+}
+
+func (c *MonitorExportCommand) Help() string {
+	helpText := `
+Usage: nomad monitor export [options]
+
+Use the 'nomad monitor export' command to export an agent's historic data
+from journald or its Nomad log file. If exporting journald logs, you must
+pass '-service-name' with the name of the nomad service.
+The '-logs-since' and '-follow' options are only valid for journald queries.
+You may pass a duration string to the '-logs-since' option to override the
+default 72h duration. Nomad will accept the following time units in the
+'-logs-since' duration string:"ns", "us" (or "µs"), "ms", "s", "m", "h".
+The '-follow=true' option causes the agent to continue to stream logs until
+interrupted or until the remote agent quits. Nomad only supports journald
+queries on Linux.
+
+If you do not use Linux or you do not run Nomad as a systemd unit, pass the
+'-on-disk=true' option to export the entirety of a given agent's nomad log file.
+
+When ACLs are enabled, this command requires a token with the 'agent:read'
+capability.
+
+
+General Options:
+
+  ` + generalOptionsUsage(usageOptsDefault|usageOptsNoNamespace) + `
+
+Monitor Specific Options:
+
+  -node-id <node-id>
+    Sets the specific node to monitor. Accepts only a single node-id and cannot
+	be used with server-id.
+
+  -server-id <server-id>
+    Sets the specific server to monitor. Accepts only a single server-id and
+	cannot be used with node-id.
+
+  -service-name <service-name>
+    Sets the name of the nomad service, must match systemd conventions and
+	include the word 'nomad'. You may provide the full systemd file name
+	or omit the suffix. If your service name includes a '.', you must include
+	a valid suffix (e.g. nomad.client.service).
+
+  -log-since <duration string>
```

Comment:
There's a mismatch between this help text, the autocomplete, and the actual parameter name (`-logs-since`).